### PR TITLE
clone gophercon workshops

### PR DESCRIPTION
**Pull request by sean1588**
Hey Sara here is the PR I opened with the clones files and you can edit them here how you want. For each new URL you can see it by the name of the folder they are in.

## Checklist:

- [ ] I have reviewed the [style guide](https://github.com/pulumi/pulumi-hugo/blob/master/STYLE-GUIDE.md).
- [ ] If blogging, I have reviewed the [blogging guide](https://github.com/pulumi/pulumi-hugo/blob/master/BLOGGING.md).
- [ ] I have manually confirmed that all new links work.
- [ ] I added aliases (i.e., redirects) for all filename changes.
- [ ] If making css changes, I rebuilt the bundle.

### CODE REVIEW

File: themes/default/content/resources/friend-or-foe-ai-devops-developers/index.md [hunk 1]
```diff
@@ -0,0 +1,80 @@
+---
+# Name of the webinar.
+title: GopherCon
```
Comment (SaraDPH) [inline themes/default/content/resources/friend-or-foe-ai-devops-developers/index.md:3]:
```suggestion
title: AI, Friend or Foe | AI Talks for DevOps
```

File: themes/default/content/resources/friend-or-foe-ai-devops-developers/index.md [hunk 2]
```diff
@@ -0,0 +1,80 @@
...
+meta_desc: We are heading to GopherCon 2023. Meet us in-person at the Marriott Marquis in San Diego, or request a virtual 1:1 with one of our experts.
```
Comment (SaraDPH) [inline themes/default/content/resources/friend-or-foe-ai-devops-developers/index.md:4]:
```suggestion
meta_desc: Join us on November 8 in San Francisco for panel discussions about the impact of AI in DevOps and software development.
```

File: themes/default/content/resources/friend-or-foe-ai-devops-developers/index.md [hunk 3]
```diff
@@ -0,0 +1,80 @@
...
+# External webinars will link to an external page instead of a webinar
+# landing/registration page. If the webinar is external you will need
+# set the 'block_external_search_index' flag to true so Google does not index
+# the webinar page created.
+external: true
+block_external_search_index: true
+
+# The url slug for the webinar landing page. If this is an external
+# webinar, use the external URL as the value here.
+url_slug: "https://info.pulumi.com/gophercon-2023"
```
Comment (SaraDPH) [inline themes/default/content/resources/friend-or-foe-ai-devops-developers/index.md:37]:
```suggestion
url_slug: "https://info.pulumi.com/ai-talks-for-devops-software-developers"
```

File: themes/default/content/resources/friend-or-foe-ai-devops-developers/index.md [hunk 4]
```diff
@@ -0,0 +1,80 @@
...
+#     gotowebinar_key: ""
+
+# Content for the left hand side section of the page.
+main:
+    # Webinar title.
+    title: ""
+    # URL for embedding a URL for ungated webinars.
+    youtube_url: ""
+    # Sortable date. The datetime Hugo will use to sort the webinars in date order.
+    sortable_date: 2023-09-26T09:00:00-07:00
```
Comment (SaraDPH) [inline themes/default/content/resources/friend-or-foe-ai-devops-developers/index.md:59]:
```suggestion
    sortable_date: 2023-11-08T17:00:00-07:00
```

File: themes/default/content/resources/friend-or-foe-ai-devops-developers/index.md [hunk 5]
```diff
@@ -0,0 +1,80 @@
...
+    # Duration of the webinar.
+    duration: "3 days"
```
Comment (SaraDPH) [inline themes/default/content/resources/friend-or-foe-ai-devops-developers/index.md:61]:
```suggestion
    duration: "1 day"
```

**Comment by SaraDPH** [comment]
@sean1588 @cnunciato the events are at 5 pm... does it use the 24-hour format? I'm not sure how to indicate it otherwise.

File: themes/default/content/resources/friend-or-foe-ai-devops-developers/index.md [hunk 6]
```diff
@@ -0,0 +1,80 @@
...
+# External webinars will link to an external page instead of a webinar
+# landing/registration page. If the webinar is external you will need
+# set the 'block_external_search_index' flag to true so Google does not index
+# the webinar page created.
+external: true
+block_external_search_index: true
+
+# The url slug for the webinar landing page. If this is an external
+# webinar, use the external URL as the value here.
+url_slug: "https://info.pulumi.com/ai-talks-for-devops-software-developers"
```
Comment (SaraDPH) [inline themes/default/content/resources/friend-or-foe-ai-devops-developers/index.md:37]:
```suggestion
url_slug: "https://info.pulumi.com/sf-ai-talks-for-devops-software-developers"
```

File: themes/default/content/resources/friend-or-foe-ai-devops-developers/index.md [hunk 7]
```diff
@@ -0,0 +1,80 @@
+---
+# Name of the webinar.
+title: AI, Friend or Foe | AI Talks for DevOps
+meta_desc: Join us on November 8 in San Francisco for panel discussions about the impact of AI in DevOps and software development.
```
Comment (SaraDPH) [inline themes/default/content/resources/friend-or-foe-ai-devops-developers/index.md:4]:
```suggestion
meta_desc: Join us to explore how AI impacts DevOps and how we build, deploy, and manage infrastructure and software applications. [in person | San Francisco]
```

File: themes/default/content/resources/generative-ai-apps-devops/index.md [hunk 8]
```diff
@@ -0,0 +1,80 @@
+---
+# Name of the webinar.
+title: Generative AI, Apps, and DevOps | AI/ML Talks
+meta_desc: Join us on Oct 19 at the Pulumi HQ (Seattle) - Dive into Generative AI, Apps, and DevOps through Lighting Talks and a workshop by experts in AI/ML.
```
Comment (SaraDPH) [inline themes/default/content/resources/generative-ai-apps-devops/index.md:4]:
```suggestion
meta_desc: Dive into Generative AI, Apps, and DevOps through Lighting Talks and a workshop by experts in AI/ML [in person | Seattle]
```

**Review by SaraDPH — APPROVED** [review]

File: themes/default/content/resources/friend-or-foe-ai-devops-developers/index.md [hunk 9]
```diff
@@ -0,0 +1,80 @@
+---
+# Name of the webinar.
+title: AI, Friend or Foe | AI Talks for DevOps
```
Comment (SaraDPH) [inline themes/default/content/resources/friend-or-foe-ai-devops-developers/index.md:3]:
```suggestion
title: "AI: Friend or Foe | AI Talks for DevOps"
```

**Review by SaraDPH — APPROVED** [review]

File: themes/default/content/resources/friend-or-foe-ai-devops-developers/index.md [hunk 10]
```diff
@@ -0,0 +1,80 @@
+---
+# Name of the webinar.
+title: "AI: Friend or Foe | AI Talks for DevOps"
+meta_desc: Join us to explore how AI impacts DevOps and how we build, deploy, and manage infrastructure and software applications. [in person | San Francisco]
```
Comment (cnunciato) [inline themes/default/content/resources/friend-or-foe-ai-devops-developers/index.md:4]:
```suggestion
meta_desc: Join us to explore how AI impacts DevOps and how we build, deploy, and manage infrastructure and software applications. (In-person, San Francisco)
```

File: themes/default/content/resources/generative-ai-apps-devops/index.md [hunk 11]
```diff
@@ -0,0 +1,80 @@
+---
+# Name of the webinar.
+title: Generative AI, Apps, and DevOps | AI/ML Talks
+meta_desc: Dive into Generative AI, Apps, and DevOps through Lighting Talks and a workshop by experts in AI/ML [in person | Seattle]
```
Comment (cnunciato) [inline themes/default/content/resources/generative-ai-apps-devops/index.md:4]:
```suggestion
meta_desc: Dive into Generative AI, Apps, and DevOps through Lighting Talks and a workshop by experts in AI/ML (In-person, Seattle)
```

**Review by cnunciato — APPROVED** [review]
Couple of suggestions, but otherwise 👍

**Pull request merged**